package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "quarkify",
-  "version": "1.18.3",
+  "version": "1.18.4",
```

Comment:
adding a new component means a whole new subversion.
1.19.0 is preferred

Reply:
Yes, it should be 1.19.0